ui/screens.py

```diff
@@ -1405,6 +1415,120 @@ def poolAuthed(self, res):
         window.stack(self)
 
 
+class StatsScreen(Screen):
+    """
+    A screen that shows network and ticket stats.
+    """
+
+    def __init__(self, app):
+        """
+        Args:
+            app (TinyDecred): The TinyDecred application instance.
+        """
+        super().__init__(app)
+        self.isPoppable = True
+        self.canGoHome = True
+
+        self.liveTickets = {}
+
+        self.app.registerSignal(ui.SYNC_SIGNAL, self.setStats)
+
+        self.wgt.setMinimumWidth(400)
+        self.wgt.setMinimumHeight(225)
+
+        lbl = Q.makeLabel("Stats", 18)
+        self.layout.addWidget(lbl, 0, Q.ALIGN_LEFT)
+
+        lbl = Q.makeLabel("current height:", 16)
+        h = self.height = Q.makeLabel("", 16)
+        wgt, _ = Q.makeSeries(Q.HORIZONTAL, lbl, h)
+        self.layout.addWidget(wgt)
+
+        # A row of voted and revoked ticket numbers.
+        lbl = Q.makeLabel("voted: ", 16)
+        lbl2 = Q.makeLabel(", revoked: ", 16)
+        qty = self.votedCount = Q.makeLabel("", 16)
+        qty2 = self.revokedCount = Q.makeLabel("", 16)
+        wgt, _ = Q.makeSeries(Q.HORIZONTAL, lbl, qty, lbl2, qty2)
+        self.layout.addWidget(wgt)
+
+        # A row of immature, live, and revocable ticket numbers.
+        lbl = Q.makeLabel("immature: ", 16)
+        lbl2 = Q.makeLabel(", live: ", 16)
+        lbl3 = Q.makeLabel(", revocable: ", 16)
+        qty = self.immatureCount = Q.makeLabel("", 16)
+        qty2 = self.liveCount = Q.makeLabel("", 16)
+        qty3 = self.revocableCount = Q.makeLabel("", 16)
+        wgt, _ = Q.makeSeries(Q.HORIZONTAL, lbl, qty, lbl2, qty2, lbl3, qty3)
+        self.layout.addWidget(wgt)
+
+        lbl = Q.makeLabel("total staked:", 16)
+        amt = self.ticketValue = Q.makeLabel("", 16)
+        wgt, _ = Q.makeSeries(Q.HORIZONTAL, lbl, amt)
+        self.layout.addWidget(wgt)
+
+        # List of live tickets. Click to open dcrdata in browser.
+        wgt = self.ticketsWgt = QtWidgets.QListWidget()
+        wgt.itemClicked.connect(self.onTicketItemClicked)
+        self.layout.addWidget(wgt)
+
+    def stacked(self):
+        """
+        stacked is called on screens when stacked by the TinyDialog.
+        """
+        pass
```

Comment:
Unnecessary because of 

https://github.com/JoeGruffins/tinydecred/blob/286d7aa13dffe635f32feb5681df5b2e2e380844/ui/screens.py#L176

but if you'd prefer, you can eliminate that check, and move this `pass`ing method to `Screen`.